src/contracts/ethr-status-registry.json

```diff
@@ -0,0 +1,84 @@
+[
+  {
+    "constant": true,
+    "inputs": [
+      {
+        "name": "",
+        "type": "bytes32"
+      },
+      {
+        "name": "",
+        "type": "address"
+      }
+    ],
+    "name": "revocations",
+    "outputs": [
+      {
+        "name": "",
+        "type": "uint256"
+      }
+    ],
+    "payable": false,
+    "stateMutability": "view",
+    "type": "function"
+  },
+  {
+    "anonymous": false,
+    "inputs": [
+      {
+        "indexed": false,
+        "name": "revoker",
+        "type": "address"
+      },
+      {
+        "indexed": false,
+        "name": "digest",
+        "type": "bytes32"
+      }
+    ],
+    "name": "Revoked",
+    "type": "event"
+  },
+  {
+    "constant": false,
+    "inputs": [
+      {
+        "name": "digest",
+        "type": "bytes32"
+      }
+    ],
+    "name": "revoke",
+    "outputs": [
+      {
+        "name": "",
+        "type": "bool"
+      }
+    ],
+    "payable": false,
+    "stateMutability": "nonpayable",
+    "type": "function"
+  },
+  {
+    "constant": true,
+    "inputs": [
+      {
+        "name": "party",
+        "type": "address"
+      },
+      {
+        "name": "digest",
+        "type": "bytes32"
+      }
+    ],
+    "name": "revoked",
+    "outputs": [
+      {
+        "name": "",
+        "type": "bool"
+      }
+    ],
+    "payable": false,
+    "stateMutability": "view",
+    "type": "function"
+  }
```

Comment:
You only need this part of the ABI. The rest can be safely be deleted

Reply:
Good catch, thank you